voteswap/templates/base.html

```diff
@@ -47,9 +49,14 @@
       </div>
     </footer>
 
-    <!-- jQuery (necessary for Bootstrap's JavaScript plugins) -->
-    <script src="{% static 'jquery.slim.min.js' %}"></script>
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.1.0/jquery.min.js"></script>
```

Comment:
Yeah, vendor it please. App Engine does some optimization of delivering static assets, IIRC.